src/constants.ts

```diff
@@ -17,21 +17,16 @@
 // the appropriate flags. However, just because we have the API does not
 // guarantee that AR will work.
 export const HAS_WEBXR_DEVICE_API = navigator.xr != null &&
-    self.XRSession != null && self.XRDevice != null &&
-    self.XRDevice.prototype.supportsSession != null;
+    self.XRSession != null && navigator.xr.supportsSession != null;
 
 export const HAS_WEBXR_HIT_TEST_API =
     HAS_WEBXR_DEVICE_API && self.XRSession!.prototype.requestHitTest;
 
-export const HAS_FULLSCREEN_API = document.documentElement != null &&
-    document.documentElement.requestFullscreen != null;
```

Comment:
I think there's some more fullscreen stuff that hasn't been reverted yet.

Reply:
This constant only appeared to be used for WebXR mode. It was used in utilities.ts's assertIsArCandidate which is only used in the WebXR ARRenderer.ts, and I had removed it there since immersive-ar mode doesn't require use of the Fullscreen API. (I think that was part of the reason I was under the impression that the other AR code didn't need fullscreen.)